### DISCUSSION TITLE
Security review

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/EdgeCloud#238] [General]
Considering: https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Readiness-Checklist.md

We adopted OpenId, so we can declare in our Readiness-Checklist file that step 7 is "Approved".

https://github.com/camaraproject/CallForwardingSignal/blob/main/documentation/API_documentation/CFS-API-Readiness-Checklist.md

Do you agree?

**Comment by javierlozallu** [comment]
I agree, but as the Readiness-Checklist is being changed we can close this discussion and open a new one to discuss the new requirements because they are very different from the previous one.

**Comment by FabrizioMoggio** [comment]
I agree
